sourcetracker/_sourcetracker.py

```diff
@@ -795,6 +949,15 @@ def f(cp_and_sink):
                                               sinks.index, sources.index,
                                               sources.columns,
                                               create_feature_tables, loo=False)
+
+    # Collate convergence time results
+    if autoc is not None:
+        cols = ['r_%i' % i for i in range(restarts)]
+        convs = pd.DataFrame(np.array([i[3] for i in results]),
+                             index=sinks.index, columns=cols)
+        convs['converging_fraction'] = [i[4] for i in results]
+        return mpm, mps, fas, convs
```

Comment:
Same comment as above about not changing the number of values returned from the function. I *highly* recommend not going with this approach.